_author_instructions/01_policies.md

```diff
@@ -103,6 +103,31 @@ However, it is still important that your references use Title Case for article t
 You are responsible for checking that any special characters appear properly, and otherwise ensuring that your final reference list is correct and well-formatted, including sufficient detail.
 Pay particular attention to formatting of non-journal articles as it is easy to end up omitting key citation information for content like software, poster presentations, preprints or other materials.
 
+### Figures
+
+Our largest concern in the construction and preparation of figures is their quality in the final PDF, so we do not *require* any specific image format for figures.
+Rather, figures should:
+- Appear polished in the final PDF of the article, even if zoomed in a great deal
+- Be of sufficiently high quality that they look good printed, in both color and grayscale (which has implications for color choices)
+- Be stylistically consistent with the rest of the paper (ideally using OpenSans as the font, and having similar font sizes/styles to the rest of the paper)
+- Use words, symbols, and units for axis labels, *e.g.* not just a plot of sigma vs. r but words should explain what sigma and r are, and what units they use
+- Use self-contained captions so that the point of the figure can be understood without referring to details in the text
+- Not be so large that the final PDF is a huge file
```

Comment:
Define huge file, such as less than 5 MB, though most papers will be closer to 1-2 MB, and some documents  that are inherently graphics heavy may need to be larger.  And I'm open on the exact numbers.

Reply:
I'd say no more than 10 MB (typical e-mail attachment size) if you want to get concrete about it.